controller/src/main/java/io/pravega/controller/store/stream/OperationNotAllowed.java

```diff
@@ -0,0 +1,30 @@
+/**
+ * Copyright (c) 2017 Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.pravega.controller.store.stream;
+
+import io.pravega.controller.retryable.RetryableException;
+
+/**
+ * Exception thrown attempting an operation that is not allowed to be performed at the moment.
+ */
+public class OperationNotAllowed extends RuntimeException implements RetryableException {
```

Comment:
Instead of creating new exceptions for each error condition, shall we start consolidating these errors as specific error values of the `error` field in `StoreException`?